demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe this cell could have a heading and/or description before it, since at the moment this is 3 code cells in a row. (Especially useful if the <code>embeddings_test</code>  computation moves to be close to here too)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I've moved `embeddings_test` here and gave it a brief title/description 👍